CMakeLists.txt

```diff
@@ -38,101 +38,56 @@ if(NOT DEFINED CMAKE_INSTALL_CMAKEDIR)
     set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/nanopb")
 endif()
 
-find_package(Python REQUIRED)
```

Comment:
We could keep the find_package on python or even python3 if we want to make sure the user has python3 installed on his system